metadata/schema/streams_schema.md

```diff
@@ -0,0 +1,82 @@
+# Title
+
+Utah Streams NHD
+
+## ID
+
+b05ae129-2047-464e-bac4-3edf3a16cd9a
+
+### permanent_identifier
+
+<!--- No definition for this field. -->
+
+### fdate
+
+The date the feature was added to the dataset.
+
+### resolution
+
+<!--- No definition for this field. -->
```

Comment:
This has to do with scale. Like 1:24,000

Reply:
In the feature class, the resolution field is set to "2" for each of the features. Do we know what the 2 refers to? I was thinking it had to do with this being the second iteration of the NHD (which is high-resolution, whereas the first one I believe was 1:100,000).

Reply:
@ZachBeck do you know?

Reply:
It's a coded domain, 2 = High